docs/_articles/en/user-guide/development-models.md

```diff
@@ -95,13 +95,13 @@ When you deploy the code changes, the local version of the source files overwrit
 
 ![Deploy source to org](./images/deploy-source-to-org.png)
 
-You can deploy source of a manifest, a source file, a directory, or a folder:
+You can deploy source of a manifest, source files, directories, or folders:
 
 - Manifest
   - In VS Code explorer or editor, right-click a manifest file and select **SFDX: Deploy Source in Manifest in Org**.
   - With a manifest file open in the editor, open the Command Palette and run **SFDX: Deploy Source in Manifest in Org**.
 - Source File or Directory
-- In VS Code explorer, right-click a source file or a directory and select **SFDX: Deploy Source to Org**.
+- In VS Code explorer, right-click single or multi-selected source files or directories and select **SFDX: Deploy Source to Org**.
```

Comment:
How does the user select multiple files, is it through right-click?

Reply:
the standard explorer way -- shift select.